src/Volume.ts

```diff
@@ -130,7 +130,7 @@ export default class Volume {
   public imageInfo: ImageInfo;
   public loadSpec: LoadSpec;
   public loader?: IVolumeLoader;
-  private loadSpecRequired: LoadSpec;
+  private loadSpecRequired: Required<LoadSpec>;
```

Comment:
What do you think of a comment up here saying why loadSpecRequired is different than loadSpec?  Just to show the intention...